metasv/generate_final_vcf.py

```diff
@@ -70,12 +123,24 @@ def convert_metasv_bed_to_vcf(bedfile=None, vcf_out=None, vcf_template_file=vcf_
                 index_to_use = sub_types.index("DEL")
                 svmethods_s = set(svmethods) - {"SC","AS"}
                 is_pass = len(svmethods_s) > 1
+                if "AS" in svmethods:
+                    pos = int(interval.fields[6])
+                    end = int(interval.fields[7])
+                    svlen = int(interval.fields[8])
+                    is_pass = (int(interval.fields[8]) != -1) and (svlen >= 100)
             elif "INV" in sub_types:
                 index_to_use = sub_types.index("INV")
-                svmethods_s = set(svmethods) - {"SC"}
+                svmethods_s = set(svmethods) - {"SC","AS"}
                 is_pass = len(svmethods_s) > 1
-            elif "INS" in sub_types and "SC" in sub_methods:
-                index_to_use = sub_methods.index("SC")
+                if "AS" in svmethods:
+                    pos = int(interval.fields[6])
```

Comment:
`pos, end, svlen = map(int, interval.fields[6:9])`? and `is_pass = svlen >= 100`?